@kiva/kv-components/vue/KvLendCta.vue

```diff
@@ -206,6 +206,18 @@ export default {
 			type: Boolean,
 			default: true,
 		},
+		primaryButtonText: {
+			type: String,
+			default: 'Lend',
+		},
+		secondaryButtonText: {
+			type: String,
+			default: 'Checkout now',
+		},
+		secondaryButtonHandler: {
```

Comment:
What is the use case for this "alternative checkout functionality"? Could we use an emit here instead?

Reply:
We use this to emit an event that removes the loan from the basket.

Reply:
https://www.figma.com/design/fvwjgrVzEEKCskC4Cn2cI3/Impact-Dashboard-page-v2---Kiva%40Work---Q124---SH?node-id=924-4785&t=8hgxzot9dgCvOcMW-0

Reply:
My first impression was to also use an emit here but I also see that we're passing in a function for several other features so we've kind of mixed the approaches into this component... Either way we have to "handle" the operation in the parent...so I think it's fine for now.

@michelleinez Please keep this in mind once you get to integrating with the [LoanCardContainer](https://github.com/kiva/cms-page-server/blob/main/components/loan/cards/KvClassicLoanCardContainer.vue) in CPS...it may be that it's easier to manage the email approach...

Reply:
Using an emit here instead of passing in the function wouldn't be able to override the existing functionality when the function is defined since the data is only one way from child to parent.  We need to alter child state as well if the function is defined to not go through the checkout process.

Reply:
It's true we'd need to also conditionally stop the checkout action. We'd need to refactor the `externalLinks` prop usage to stop the navigation...

Looking at the current button usage which has a `@click.native` handler we may also need to include `preventDefault()` in the handler function we pass in to keep the click from going out to checkout...